src/main/java/CinemaRoomRestService/RestService/domain/entity/Statistics.java

```diff
@@ -0,0 +1,25 @@
+package CinemaRoomRestService.RestService.domain.entity;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Getter
+public class Statistics {
+    private int currentIncome;
+    private int availableSeats;
+    private int purchasedTickets;
+
+    @Autowired
+    public Statistics(List<Seat> list) {
```

Comment:
list is not a good naming seats is better on my mind

Reply:
Fixed